src/test/java/com/make/my/day/hm2/Homework02.java

```diff
@@ -218,17 +228,17 @@ public LazyProperty(Supplier<Integer> supplier) {
 
     //TODO: initialize "lazy" using "supplier" only one time
     public int getLazy() {
-      return 0;
+      return lazy = supplier.get();
```

Comment:
That's wrong, every time when we will use `getLazy()` it provides us new value from the supplier.

You need `initialize "lazy" using "supplier" only one time`

Reply:
Fixed, homework 3 was added too